src/providers/trustless-work.provider.tsx

```
@@ -0,0 +1,30 @@
+"use client"; // make sure this is a client component
+
+import React from "react";
+import {
+  // development environment = "https://dev.api.trustlesswork.com"
+  development,
+
+  // mainnet environment = "https://api.trustlesswork.com"
+  mainNet,
```

ESLint

[error] 'mainNet' is defined but never used.

(@typescript-eslint/no-unused-vars)